packages/datetime/src/_daterangeinput.scss

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../core/src/common/variables";
+
+/*
+Date range input
+
+@react-example DateRangeInputExample
+
+Styleguide components.datetime.daterangeinput
+*/
+
+.pt-daterangeinput {
+  position: relative;
+}
+
+.pt-daterangeinput-icon-wrapper {
```

Comment:
Talked offline, let's get rid of it. It's an awkward toggle that we've had in `DateInput` through the `openOnFocus` prop, but it's not useful since the date input doesn't do anything more than a regular input